ui/log_page.go

```diff
@@ -0,0 +1,109 @@
+package ui
+
+import (
+	"sync"
+
+	"gioui.org/layout"
+	"gioui.org/widget"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+const PageLog = "Log"
+
+type logPage struct {
+	theme *decredmaterial.Theme
+
+	copyLog  *widget.Clickable
+	copyIcon *widget.Image
+
+	entriesList layout.List
+	fullLog     string
+	logEntries  []decredmaterial.Label
+	entriesLock sync.Mutex
+}
+
+func (win *Window) LogPage(common pageCommon, internalLog chan string) layout.Widget {
+	pg := &logPage{
+		theme: common.theme,
+		entriesList: layout.List{
+			Axis:        layout.Vertical,
+			ScrollToEnd: true,
+		},
+		copyLog:    new(widget.Clickable),
+		logEntries: make([]decredmaterial.Label, 0, 20),
+	}
+
+	pg.copyIcon = common.icons.copyIcon
+	pg.copyIcon.Scale = 0.25
+
+	go pg.watchLogs(internalLog)
+
+	return func(gtx C) D {
+		//pg.handle(common)
+		return pg.Layout(gtx, common)
+	}
+}
+
+func (pg *logPage) copyLogEntries(common pageCommon) {
+	go func() {
+		pg.entriesLock.Lock()
+		defer pg.entriesLock.Unlock()
+		common.clipboard <- WriteClipboard{
+			Text: pg.fullLog,
+		}
+	}()
+}
+
+func (pg *logPage) watchLogs(internalLog chan string) {
+	for l := range internalLog {
+		entry := l[:len(l)-1]
+		pg.entriesLock.Lock()
+		pg.fullLog += l
+		pg.logEntries = append(pg.logEntries, pg.theme.Body1(entry))
+		pg.entriesLock.Unlock()
+	}
+}
+
+func (pg *logPage) Layout(gtx C, common pageCommon) D {
+	container := func(gtx C) D {
+		page := SubPage{
+			title: "Wallet log",
+			back: func() {
+				*common.page = PageDebug
+			},
+			extraItem: pg.copyLog,
+			extra: func(gtx C) D {
+				return layout.Center.Layout(gtx, func(gtx C) D {
+					return layout.Inset{Top: values.MarginPadding7}.Layout(gtx, func(gtx C) D {
+						return decredmaterial.Clickable(gtx, pg.copyLog, func(gtx C) D {
+							return pg.copyIcon.Layout(gtx)
```

Comment:
```suggestion
sz := gtx.Constraints.Max.X
pg.copyIcon.Scale = float32(sz) / float32(gtx.Px(unit.Dp(float32(sz))))
return pg.copyIcon.Layout(gtx)
```
This will make the icon more visible